build_tools/cmake/iree_run_module_test.cmake

```diff
@@ -181,6 +190,13 @@ function(iree_run_module_test)
     set(_TEST_XFAIL TRUE)
   endif()
 
+  # Android test runs on device and needs to set the file location properly.
+  if(ANDROID)
+    cmake_path(GET _RULE_MODULE_SRC FILENAME _SRC_FILE_BASE)
+    set(_SRC_RELATIVE_PATH "${_PACKAGE_PATH}/${_RULE_NAME}/${_SRC_FILE_BASE}")
+    set(_OUTPUT_FLAGFILE "${_PACKAGE_PATH}/${_RULE_NAME}/${_OUTPUT_FLAGFILE}")
```

Comment:
Can you move these to the places where those variables defined (line 168 and line 104)

Reply:
Done.